components/brave_news/browser/test/BUILD.gn

```diff
@@ -40,9 +40,14 @@ source_set("brave_news_unit_tests") {
     "//brave/components/l10n/common",
     "//brave/components/l10n/common:test_support",
     "//brave/components/time_period_storage",
-    "//chrome/browser",
-    "//chrome/test:test_support",
+    "//components/history/core/browser",
```

Comment:
did gn check ask for all of these? Seems odd because I don't see some of them added as includes

Reply:
This is probably for the `suggestions` or `feed_v2_builder` unittest

But yeah, `gn_check` asked for these.

Reply:
huh, I wonder why it wasn't complaining about them before, but I guess I do see these in there